subsys/bluetooth/host/classic/shell/bredr.c

```diff
@@ -36,16 +36,23 @@
 static struct bt_conn *pairing_conn;
 #endif /* CONFIG_BT_CONN */
 
-#define DATA_BREDR_MTU		48
+#if defined(CONFIG_BT_L2CAP_RET) || defined(CONFIG_BT_L2CAP_FC) ||                                 \
+	defined(CONFIG_BT_L2CAP_ENH_RET) || defined(CONFIG_BT_L2CAP_STREAM)
+#define L2CAP_BR_RET_FC_ENABLE 1
+#else
+#define L2CAP_BR_RET_FC_ENABLE 0
+#endif
```

Comment:
This kind of construction sounds like those four Kconfig options should just select the same virtual option - then you don't need to do this additional stuff in the code itself.

Reply:
Yes. Added a virtual config.